packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.service.js

```diff
@@ -55,6 +58,15 @@ export default class BreadcrumbService {
                 );
               }
             });
+
+            if (breadcrumbUrlResolvable) {
+              transition
+                .injector(state.name)
+                .getAsync('breadcrumbUrl')
+                .then((url) => {
+                  entry.url = url;
+                });
+            }
```

Comment:
**question:** Can you explain why the need to introduce new resolvable `breadcrumbUrl`?

Reply:
States are used to automatically create Breadcrumbs levels.
For uapp responding with url like '/public-cloud/pci/projects/<id>/<uapp>' the first level of breadcrumb will be '/uapp' by default. But with this fix, we can tell to breadcrumb that the first level should be '/public-cloud/pci/projects/<id>'